test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectHandlerTests.cs

```diff
@@ -443,12 +275,12 @@ private static TestServer CreateServer(IOptions<OpenIdConnectAuthenticationOptio
             );
         }
 
-        private static TestServer CreateServer(CustomConfigureOptions configureOptions, ILoggerFactory loggerFactory)
+        private static TestServer CreateServer(CustomConfigureOptions configureOptions, IUrlEncoder encoder, ILoggerFactory loggerFactory, OpenIdConnectAuthenticationHandler handler = null)
```

Comment:
The testing here is at the level of UnitTest / Functional where I want tight control over exactly the flow in each method.